unit-tests/live/syncer/test-throughput.cpp

```diff
@@ -0,0 +1,248 @@
+        {
+            auto actual_fps = frames.front().second;
+            float prev_frame_time = frames.front().first;
+            frames.erase(frames.begin());
+            float expected_dt_ms = 1000/actual_fps; // convert sec -> msec
+            
+            for (auto& frame : frames)
+            {
+                auto frame_time = frame.first;
+                float calc_dt_msec = ((float)frame_time - prev_frame_time)/1000; //convert usec -> msec
+                float dt_msec_ratio = calc_dt_msec / expected_dt_ms;
+                CAPTURE(expected_dt_ms, calc_dt_msec, dt_msec_ratio, frame_time, prev_frame_time);
+                CHECK(dt_msec_ratio > 0.9);
+                prev_frame_time = frame_time;
+            }
+        }
+
+        sensor _rgb_sensor;
+        sensor _ir_sensor;
+        std::vector<rs2::stream_profile > _rgb_stream_profile;
+        std::vector<rs2::stream_profile > _ir_stream_profile;
+        std::vector <std::pair<float, float>> _ir_frames_arrival_info; //{HW timestamp, fps}
+        std::vector <std::pair<float, float>> _rgb_frames_arrival_info; //{HW timestamp, fps}
+        std::mutex _mutex;
+        rs2::syncer _sync;
+        float _delta; // dt in seconds that took first frames (before setting exposure) to arrive
+    };
+
+    // Require at least one device to be plugged in
+    rs2::context ctx;
+    auto list = ctx.query_devices();
+    REQUIRE(list.size());
+    auto dev = list.front();
+    auto sensors = dev.query_sensors();
+    REQUIRE(dev.supports(RS2_CAMERA_INFO_PRODUCT_LINE));
+    std::string device_type = list.front().get_info(RS2_CAMERA_INFO_PRODUCT_LINE);
+    if (device_type != "D400")
+    {
+        std::cout << "This test runs only with D400 device!"<< std::endl;
+        exit(EXIT_SUCCESS);
+    }
+    int width = 848;
+    int height = 480;
+
+    // extract IR/RGB sensors
+    sensor rgb_sensor;
+    sensor ir_sensor;
+    std::vector<rs2::stream_profile > rgb_stream_profile;
+    std::vector<rs2::stream_profile > ir_stream_profile;
+    for (auto& s : sensors)
+    {
+        auto info = std::string(s.get_info(RS2_CAMERA_INFO_NAME));
+        auto stream_profiles = s.get_stream_profiles();
+        for (auto& sp : stream_profiles)
+        {
+            auto vid = sp.as<rs2::video_stream_profile>();
+            if (!(vid.width() == width && vid.height() == height && vid.fps() == 60)) continue;
+            if (sp.stream_type() == RS2_STREAM_COLOR && sp.format() == RS2_FORMAT_RGB8)
+                rgb_stream_profile.push_back(sp);
+            if (sp.stream_type() == RS2_STREAM_INFRARED && sp.format() == RS2_FORMAT_Y8)
+                ir_stream_profile.push_back(sp);
+        }
+        if (info == "RGB Camera")
+            rgb_sensor = s;
+
+        if (info == "Stereo Module")
+            ir_sensor = s;
+    }
+    rgb_sensor.set_option(RS2_OPTION_GLOBAL_TIME_ENABLED, 0);
+    ir_sensor.set_option(RS2_OPTION_GLOBAL_TIME_ENABLED, 0);
```

Comment:
1. test if IR/RGB sensors are valid ( I recommend to move it to lines 233/236 accordingly)
2. Test that the global TS option is supported before toggle off